CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## [Unreleased]
 
 - OIDC token request to not flow during explain
+- Added support for auto-retry for source table. Auto retry on IOException and user-defined http codes - parameter `gid.connector.http.source.lookup.retry-codes`.
```

Comment:
1. Please rebase with main. I merged "Bump development version after release of 0.19.0 " so this changelog entries should be in different section.

2. We introduce backward incompatible changes. We need to inform about that explicitly and probably explain how to migrate with some examples.